src/link/output.cpp

```diff
@@ -149,6 +147,22 @@ static void coverOverlayBanks(uint32_t nbOverlayBanks) {
 	}
 }
 
+static uint8_t getNextFillByte() {
+	if (overlayFile) {
+		int c = getc(overlayFile);
+		if (c != EOF) {
+			return c;
+		}
+	}
+
+	static bool warnedNoPadValue = false;
+	if (overlayFile && !hasPadValue && !warnedNoPadValue) {
+		warnx("Overlay file specified without pad value");
```

Comment:
That's not quite correct, because this warning is not emitted if the ROM does not go past the overlay file.

Thus, I suggest this message instead:

```suggestion
		warnx("Gone past overlay file, but no padding value was specified");
```

Also, this warning should be moved to inside of the `if (overlayFile)`, to simplify the logic a bit.